src/components/button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+export default function CountClicks(){
```

Comment:
I prefer to have any export at the bottom.

better delcare your component.

const ClickerCounter = ()=>



export default ClickerCounter

export {
anyOtherFunctions
}

Reply:
did it